numpy/lib/_histograms_impl.py

```
@@ -383,6 +383,17 @@ def _get_bin_edges(a, bins, range, weights):
     # parse the overloaded bins argument
     n_equal_bins = None
     bin_edges = None
+    if symlog:
+        if np.ndim(bins) != 0:
+            warnings.warn(
```

This should raise an error